.github/workflows/ci.yml

```diff
@@ -21,8 +37,10 @@ jobs:
         with:
           stable: true
 
-      - name: Checkout ${{ github.event.repository.name }}
+      - name: Checkout ${{ github.repository }}
         uses: actions/checkout@v3
+        with:
+          persist-credentials: false
```

Comment:
In order to add the linter for CFF, I had to touch the CI, anyway, so I also adjusted some points to improve the workflow.  *These* changes are:

- relying on a constant rather than a web hook; the constant is always available whilst the web hook might fail, so the data would be missing.
- prevent saving the GitHub Access Token of the workflow run on the runner; that would be only required for pushing changes back to GitHub.